include/micm/solver/backward_euler.inl

```diff
@@ -195,12 +189,12 @@ namespace micm
           H *= 2.0;
         }
       }
-
       // Don't let H go past the time step
       H = std::min(H, time_step - t);
     }
```

Comment:
it will be great if we could add some comments after these `}` to indicate which loop/if-else block it ends.

Reply:
I'm not sure that this is a style we want to support. Code folding is more useful than comments, I feel, since it allows whomever is reading to choose which parts of they control flow they want to hide